.github/ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,72 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as **Ready** in the CMS Design System.
+
+## Documentation is complete
+
+**What to check:**
+- [ ] Description of component or pattern
+- [ ] HTML code snippet example(s)
+- [ ] React code snippet example(s)
+- [ ] All react properties are explained
+- [ ] General guidance or best practices
+- [ ] Usability - when to use
+- [ ] Usability - when to consider something else
+- [ ] Accessibility considerations
+
+## Validate HTML
+If you are using Google Chrome or Firefox consider getting the [axe plugin](https://www.deque.com/axe) for validation.
+**What to check:**
+- If testing on W3.org
```

Comment:
What do you think about making this bullet point an action statement like `Test HTML`, and then offer several checkbox options for testing. I found a highly rated checker that users could download locally for Firefox or Chrome, with links inline. I think the checkboxes for this bullet could be something like:

* Go to [validator.w3.org](https://validator.w3.org/), paste your code in and make sure it validates without error
* Verify locally with [HTML Validator for Firefox](https://addons.mozilla.org/en-US/firefox/addon/html-validator/
* Verify locally with [HTML Validator for Chrome](https://chrome.google.com/webstore/detail/html-validator/mpbelhhnfhfjnaehkcnnaknldmnocglk/related)
* Verify locally with another checker

The ask on this bullet point is to make sure HTML is valid, then with the next point install and verify with aXe. The overlap between these tools is ~40-50%, so each has a purpose in the validation step.

Reply:
Do we want folks to complete all of those bullet points or this is a combo of the first bullet point and any of the last three?